src/styles/blocks/page.scss

```
@@ -0,0 +1,29 @@
+.page {
+  font-family: "Manrope", sans-serif;
```

Remember to use fallback fonts
```suggestion
  font-family: "Manrope", "Helvetica", sans-serif;
```